specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2017-10-01/examples/ImportImageByManifestDigest.json

```diff
@@ -0,0 +1,25 @@
+{
+    "parameters": {
+        "api-version": "2017-10-01",
+        "subscriptionId": "00000000-0000-0000-0000-000000000000",
+        "resourceGroupName": "myResourceGroup",
+        "registryName": "myRegistry",
+        "parameters": {
+            "source": {
+                "resourceId": "/subscriptions/10000000-0000-0000-0000-000000000000/resourceGroups/sourceResourceGroup/providers/Microsoft.ContainerRegistry/registries/sourceRegistry",
+                "sourceImage": "sourceRepository@sha256:0000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
Repository name has to be lower case so I would lower case the example name `sourceRepository` here to make it more realistic. The same applies to other cases.

Reply:
@djyou @nathana1 oh, note that you could model that restriction using the `pattern` property of Swagger (or JSON schema, for that matter) that allows you to specify a regex. Maybe this is a good opportunity for adding a `pattern` on `ImportSource.sourceImage`?